app/models/user.rb

```diff
@@ -218,10 +218,9 @@ def student_team
 
   private
 
-  # Ensures that the location column either contains non-whitespace text, or is NULL
```

Comment:
It would be better to leave some comment (normalisation to de-duplicate + NULL for sorting)